src/biogeophys/EnergyFluxType.F90

```diff
@@ -21,6 +21,9 @@ module EnergyFluxType
   type, public :: energyflux_type
 
      ! Fluxes
+!scs
```

Comment:
Need to remove comment lines like this, and other similar comments (scs comments and other commented-out lines).

@swensosc and @olyson - for the future, it makes code reviews easier if you can remove these commented-out things and commented initials before submitting the PR.